src/DynamoCore/Graph/Workspaces/UndoRedo.cs

```diff
@@ -422,6 +455,14 @@ public void CreateModel(XmlElement modelData)
                     OnConnectorAdded(connector); // Update view-model and view.
                 }
             }
+            else if (typeName.Contains("ConnectorPinModel"))
```

Comment:
Feel free to use nameof(ConnectorPinModel) instead of hardcoded string, so in the future auto renamed type will still work. In general, we try to avoid hardcoded string, especially type names

Reply:
Good catch! Yeah been trying to adhere to that everywhere :).